python-checks/src/test/resources/checks/nonCallableCalled.py

```diff
@@ -255,3 +255,8 @@ def my_rec(x):
         my_rec(True)
     finally:
         my_rec = None
+
+def call_non_callable_property():
+    e = OSError()
+    e.errno()  # FN
+
```

Comment:
I assume we should detect some new issues as well thanks member type resolutions, could we have some tests highlighting that?

I see we have a test already that highlights that reassignments are not tracked (see snippet below), but I feel we can have cases where we actually detect issues.
```
def member_access():
  my_callable = MyCallable()
  my_callable.non_callable = 42
  my_callable.non_callable() # FN
```